Nikita_Korzhavin/api/products.js

```diff
@@ -0,0 +1,16 @@
+// product используются для моделирования начинок, типов салатов и напитков
+function Product(price, calories, name) {
+  this.price = price || 0;
+  this.calories = calories || 0;
+  this.name = name;
+}
+
+Product.prototype.calculatePrice = function() {
```

Comment:
These two function are seem more like `getPrice` and `getCalories` since they do not calculate anything just return the value stored in the object

Reply:
fixed